assets/loadtest/network/main.tf

```diff
@@ -16,10 +16,4 @@ resource "google_compute_address" "proxy_ip" {
   name         = "proxy-ip"
   address_type = "EXTERNAL"
   network_tier = "PREMIUM"
-}
-
-resource "google_compute_address" "grafana_ip" {
```

Comment:
Just to ensure we are deleing Grafana resource but it should be still accessible via private ip ?